sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/intervalExpressions.scala

```diff
@@ -113,12 +114,14 @@ object ExtractIntervalPart {
 abstract class IntervalNumOperation(
     interval: Expression,
     num: Expression,
-    operation: (CalendarInterval, Double) => CalendarInterval,
-    operationName: String)
+    val checkOverflow: Boolean = SQLConf.get.ansiEnabled)
```

Comment:
hey wait. Does it mean this value can change when we transform/copy `MultiplyInterval`/`DivideInterval`?

Reply:
Isn't  `SQLConf.get`  thread-safe? or `set command` can take effect across spark sessions?